monkey/tests/unit_tests/monkey_island/cc/models/test_agent_registration_message.py

```diff
@@ -6,16 +6,18 @@
 import pytest
 
 from common import AgentRegistrationData
+from common.types import SocketAddress
 
 AGENT_ID = UUID("012e7238-7b81-4108-8c7f-0787bc3f3c10")
 PARENT_ID = UUID("0fc9afcb-1902-436b-bd5c-1ad194252484")
+SOCKET_ADDRESS = SocketAddress(ip="192.168.1.1", port="5000")
```

Comment:
re-use this in [test_handle_agent_registration.py](https://github.com/guardicore/monkey/pull/2353/files#diff-1bac01f51af89c43f7980e3e5a694e16851b967842c7863b98c739e0d4ddad96)?

Reply:
It's just an IP address. I don't think it's worth the refactoring.